lib/cartopy/tests/mpl/test_crs.py

```diff
@@ -8,12 +8,12 @@
 import pytest
 
 import cartopy.crs as ccrs
-from cartopy.mpl import _MPL_35
+from cartopy.mpl import _MPL_36
 
 
 @pytest.mark.natural_earth
 @pytest.mark.mpl_image_compare(
-    filename="igh_land.png", tolerance=0.5 if _MPL_35 else 3.6)
+    filename="igh_land.png", tolerance=0.5 if _MPL_36 else 3.6)
```

Comment:
CI fails here if the tolerance is set to 0.5 with mpl 3.5.

Reply:
I wouldn't read too much into that. Maybe the image was made with a `main` checkout at the time or something? Our tests are often brittle to these things, so this seems fine to me to bump one version chalking it up to we didn't know exactly which minimum version was needed.

Maybe we should think about testing with all versions of Matplotlib that we support though rather than only the newest and oldest.